content/docs/learning-guide/tutorials/environments/constructing-terrain.md

```diff
@@ -0,0 +1,96 @@
+---
+linkTitle: Constructing Terrain
+title: Construct terrain in Open 3D Engine
+description: Learn to construct terrain in Open 3D Engine (O3DE).
+weight: 200
+toc: true
+---
+
+## **Constructing Terrain**
+You can construct terrain via the Entity Outliner and Entity Inspector using gradient generators.
+
+**Setup**: Enable the Terrain Gem.
+
+*Note: You will need to create both a spawner area and spawnable area.*
+
+### **Define a spawnable area**
+
+1. In the **O3DE Editor**, create a new level or open an existing level.
+2. In the **Entity Outliner**, select the level.prefab.
+- ![A](/images/learning-guide/tutorials/environments/constructing-terrain/A.png)
+3. In the **Entity Inspector**, add both the **Terrain World** and **Terrain World Renderer** components.
+- ![B](/images/learning-guide/tutorials/environments/constructing-terrain/B.png)
+
+### **Define a spawnable area**
+
+1. Create a new entity (hotkey **Ctrl+Alt+N**). (This will be your **TerrainSpawner** entity.)
+- ![C](/images/learning-guide/tutorials/environments/constructing-terrain/C.png)
+2. With the new entity selected, in the Entity Inspector, add a **Terrain Layer Spawner** and an **Axis Aligned Box Shape** component. (You can adjust the dimensions of the Aligned Box Shape component to encompass or intersect with the Terrain Layer Spawner.)
+3. Next, add a **Terrain Heigh Gradient List** component.
+- ![D](/images/learning-guide/tutorials/environments/constructing-terrain/D.png)
+4. In the Entity Outliner, create a second entity. (This will contain your **HeightfieldGradient** entity.)
+- ![E](/images/learning-guide/tutorials/environments/constructing-terrain/E.png)
+5. With the new entity selected, in the Entity Inspector, add an **Image Gradient** Component.
+6. Next, add the required **Gradient Transfer Modifier** component.
+7. Then, add a required Shape component (e.g. **Box Shape, Sphere, Shape, Axis Aligned Box Shape**).
+    Typically, this will be a Shape Reference so that we can share the same shape as the Terrain Layer Spawner, so that the image always exactly stretches across the entire spawner.
+ -  ![F](/images/learning-guide/tutorials/environments/constructing-terrain/F.png)
+8. In the Shape Reference component (if used), assign the TerrainSpawner entity.
```

Comment:
There's a step missing here where an image gets set in the Image Gradient component. A few other things worth mentioning on that step:
* Changing the sampling type from "Point" to "Bilinear" will make the heights smoother if the image resolution doesn't exactly match the terrain resolution.
* Images should have their names end in "_gsi" to help ensure they're automatically turned into lossless gradient images. Alternatively, you can right-click on the image in the Asset Browser, choose "Set Texture Settings...", and set the preset to "GSI8", "GSI16", or "GSI32", "GSI". These will produce 8-bit, 16-bit, or 32-bit or auto-bit-detection lossless gradient images. If the preset is "Albedo", it will default to lossy 8-bit values, which doesn't work well for heights.